manifests/qpid.pp

```diff
@@ -25,8 +25,9 @@
     require   => Service['qpidd'],
     logoutput => true,
   } ~>
-  qpid::bind_event { ['entitlement.created', 'entitlement.deleted', 'pool.created', 'pool.deleted', 'compliance.created']:
+  qpid::bind { ['entitlement.created', 'entitlement.deleted', 'pool.created', 'pool.deleted', 'compliance.created']:
     ssl_cert => $client_cert,
     queue    => $candlepin_event_queue,
+    exchange => 'event',
```

Comment:
You could introduce a ```$candlepin_event_exchange``` and hardcode the default there. That at least makes it easier to override later on.